### DISCUSSION TITLE
Possibility-of-Macro-Tasks-Being-Executed-Before-Paint-and-React-useEffect

### DISCUSSION THREAD

**Discussion by giscus** [lurgi/lurgi.github.io#7] [General]
# Possibility-of-Macro-Tasks-Being-Executed-Before-Paint-and-React-useEffect

https://lurgi.github.io/Development/Possibility-of-Macro-Tasks-Being-Executed-Before-Paint-and-React-useEffect

**Comment by soi-ha** [comment]
진짜 글 잘쓰시네요!
